Userland/DevTools/UserspaceEmulator/Emulator_syscalls.cpp

```diff
@@ -283,6 +285,26 @@ int Emulator::virt$profiling_disable(pid_t pid)
     return syscall(SC_profiling_disable, pid);
 }
 
+FlatPtr Emulator::virt$perf_event(int event, FlatPtr arg1, FlatPtr arg2)
+{
+    if (event == PERF_EVENT_SIGNPOST)
+        return syscall(SC_perf_event, PERF_EVENT_SIGNPOST, arg1, arg2);
+    return 0;
```

Comment:
Should we return `-ENOSYS` in these cases?

Reply:
Sure!